From c10a401686105da035ba5336f3a8d17844c6a2ee Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 12 Aug 2008 10:36:49 +0800
Subject: [PATCH] ENGR00087437 Fix MX31 compile error

Add dumy function ipu_disp_direct_write and
ipu_init_async_panel to ipu_adc.c

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 drivers/mxc/ipu/ipu_adc.c |   14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

Index: linux-2.6.26/drivers/mxc/ipu/ipu_adc.c
===================================================================
--- linux-2.6.26.orig/drivers/mxc/ipu/ipu_adc.c	2009-03-14 23:35:31.000000000 +0100
+++ linux-2.6.26/drivers/mxc/ipu/ipu_adc.c	2009-03-15 01:11:08.000000000 +0100
@@ -1,5 +1,5 @@
 /*
- * Copyright 2005-2007 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2005-2008 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -670,6 +670,18 @@ static void _ipu_set_cmd_data_mappings(d
 	}
 }
 
+void ipu_disp_direct_write(ipu_channel_t channel, u32 value, u32 offset)
+{
+	/*TODO*/
+}
+
+int ipu_init_async_panel(int disp, int type, uint32_t cycle_time,
+			 uint32_t pixel_fmt, ipu_adc_sig_cfg_t sig)
+{
+	/*TODO:uniform interface for ipu async panel init*/
+	return 0;
+}
+
 EXPORT_SYMBOL(ipu_adc_write_template);
 EXPORT_SYMBOL(ipu_adc_write_cmd);
 EXPORT_SYMBOL(ipu_adc_set_update_mode);
